index.js

```diff
@@ -9,7 +9,7 @@ const gitHubInjection = cb => {
     throw new TypeError('Callback is not a function');
   }
 
-  document.addEventListener('pjax:end', cb);
+  document.addEventListener('turbo:render', cb);
```

Comment:
Both could be kept to preserve GitHub Enterprise support, but it could cause issues
```suggestion
  document.addEventListener('pjax:end', cb);
  document.addEventListener('turbo:render', cb);
```

Reply:
It's my understanding that they haven't fully moved to turbo yet and it could be switched back to the pjax implementation at any time. Turbo was put behind a feature flag and they've been switching between them as bugs are found & fixed.

Would it be better to do this with feature detection and only register one listener? There's a `window.Turbo` when it's enabled.

Reply:
Extensions don’t have access to JavaScript variables. However we could look for the `turbo-frame` element after dom-ready.

Reply:
Sorry for the delayed response. Waiting for the DOM to complete could cause other issues depending on the extension use case. What if we register both listener and use a flag variable to prevent the callback from being called twice?

Reply:
Let's merge this as is (including both event listeners). From my current understanding this should be fine as long as GitHub does not ship both versions. But even if they do, and the callback gets called twice, OctoLinker should be able to handle this.